py3status/modules/xrandr_toggle.py

```diff
@@ -88,7 +88,7 @@ def _xrandr(self):
                         monitors.append(source)
             return monitors
         except:
-            return self._error('error while parsing xrandr', None)
+            return self._error('error while parsing xrandr')
```

Comment:
This is why you should remove the try/except
On 3 Apr 2016 17:40, "Themistokle Benetatos" <notifications@github.com>
wrote:

> In py3status/modules/xrandr_toggle.py
> <https://github.com/ultrabug/py3status/pull/242#discussion_r58313152>:
>
> > @@ -88,7 +88,7 @@ def _xrandr(self):
> >                          monitors.append(source)
> >              return monitors
> >          except:
> > -            return self._error('error while parsing xrandr', None)
> > +            return self._error('error while parsing xrandr')
>
> This is still not the proper way to do this, I'm returning an error
> response instead of the monitors list.
> Sorry for the mixup, I will fix this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ultrabug/py3status/pull/242/files/574de66dd9d0c282dd5428d4928a420fde2cecf0..62c327492aec49871ee2fdce1ae500a487ecdba6#r58313152>
>